### PR TITLE
Bug#1558251 - streamlined examples in planning guide and advance install guide

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://bugzilla.redhat.com/show_bug.cgi?id=1558251

Changes in `install_config/install/advanced_install.adoc`
1. Updated the example titles to match current titles in Planning guide.

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@bfallonf peer review please 💵

File: install_config/install/advanced_install.adoc [hunk 1]
```diff
@@ -1972,6 +1972,69 @@ Moving from a single master cluster to multiple masters after installation is
...
+|*node1.example.com*
+.2+.^|Node
+
+|*node2.example.com*
+|===
+
+You can see these example hosts present in the *[masters]*, *[etcd]*, and
+*[nodes]* sections of the following example inventory file:
+
+.Single Master and Multiple Nodes Inventory File
```
Comment (gaurav-nelson) [inline install_config/install/advanced_install.adoc]: @vikram-redhat need QE review for this example file.

**Comment by bfallonf** [comment]
@gaurav-nelson LGTM (once it passes QE) :goat:

**Comment by vikram-redhat** [comment]
@bfallonf you need to remove the peer review needed label and attach the peer review done label if you are finished.

**Comment by gaurav-nelson** [comment]
@sferich888 @sdodson Please take a look. 

I have made some quick-fixes for 3.9 release. After the release I will update the docs so that both planning and inventory files use same source for these examples.

File: install_config/install/advanced_install.adoc [hunk 2]
```diff
@@ -1972,6 +1972,69 @@ Moving from a single master cluster to multiple masters after installation is
 not supported.
 ====
 
+[discrete]
+[[single-master-multi-node-ai]]
+==== Single Master and Multiple Nodes
```
Comment (ganhuang) [inline install_config/install/advanced_install.adoc]: @gaurav-nelson It makes me a little hard to identify the differences with `Single Master, Single etcd, and Multiple Nodes`.

In `Single Master and Multiple Nodes`, we just don't have `infra` nodes? (while comparing with `Single Master, Single etcd, and Multiple Nodes`).

File: install_config/install/advanced_install.adoc [hunk 3]
```diff
@@ -1972,6 +1972,69 @@ Moving from a single master cluster to multiple masters after installation is
...
+
+# Set variables common for all OSEv3 hosts
+[OSEv3:vars]
+# SSH user, this user should allow ssh based auth without requiring a password
+ansible_ssh_user=root
+
+# If ansible_ssh_user is not root, ansible_become must be set to true
+#ansible_become=true
+
+openshift_deployment_type=openshift-enterprise
```
Comment (ganhuang) [inline install_config/install/advanced_install.adoc]: Should we also consider origin like elsewhere.

```
 ifdef::openshift-enterprise[]
 openshift_deployment_type=openshift-enterprise
 endif::[]
 ifdef::openshift-origin[]
 openshift_deployment_type=origin
 endif::[]
```

File: install_config/install/advanced_install.adoc [hunk 4]
```diff
@@ -1972,6 +1972,69 @@ Moving from a single master cluster to multiple masters after installation is
...
+
+# host group for etcd
+[etcd]
+master.example.com
+
+# host group for nodes, includes region info
+[nodes]
+master.example.com
+node1.example.com
+node2.example.com
```
Comment (ganhuang) [inline install_config/install/advanced_install.adoc]: To have the infra components installed, we need to add infra labels to the nodes at least. 

```
openshift_node_labels="{'region': 'infra', 'zone': 'default'}"
```

File: install_config/install/advanced_install.adoc (hunk 2 above)
Comment (gaurav-nelson) [reply install_config/install/advanced_install.adoc]: Yup, you are right. We already have that in our docs. I will remove this section then.

File: install_config/install/advanced_install.adoc (hunk 3 above)
Comment (gaurav-nelson) [reply install_config/install/advanced_install.adoc]: I'll update other examples as well for this one.

**Comment by ganhuang** [comment]
Thanks, good to me.

**Review by sdodson — APPROVED** [review]

**Pull request merged**